addons/web/static/src/core/l10n/dates.js

```diff
@@ -349,7 +349,10 @@ export function formatDateTime(value, options = {}) {
     if (!value) {
         return "";
     }
-    const format = options.format || localization.dateTimeFormat;
+    var format = options.format || localization.dateTimeFormat;
+    if (options.showSeconds === false) {
+        format = format.split(' ')[0] + ' ' + localization.shortTimeFormat;
```

Comment:
this seems weird. I wonder if it would not be better to simply have
= `${dateFormat} ${shortTimeFormat}`

Reply:
if there is no splitting you get " " + short format with your code

Reply:
or you could get several times hours,...